### ISSUE TITLE
Eddy Touch

### ISSUE THREAD

**Issue by 0xfde9** [bigtreetech/Eddy#43]
Hello team.

Any update regarding "Eddy Touch" I'm still getting from time to time the inaccurate 'z offset' 

Regards.

**Comment by looxonline** [comment]
This is in the pipeline

**Comment by igorscheffer** [comment]
any news about eddy touch

**Comment by HeavilyModdedEnder3** [comment]
Happy New Year! Just wondering if there has been any update since September

**Comment by LeandroSQ** [comment]
+1

**Comment by ElForastero** [comment]
+1

**Comment by HeavilyModdedEnder3** [comment]
![Screenshot_20250115_193933_Chrome](https://github.com/user-attachments/assets/f893c920-4c75-44f0-baa8-a99ebd581ffb)

**Comment by looxonline** [comment]
It may be sooner than you think. A bright member of the community has helped with the work and it is going into testing.

**Comment by jbenzaquen42** [comment]
https://hackmd.io/yEF4CEntSHiFTj230CdD0Q

Looks like we can test it now!

**Comment by looxonline** [comment]
Yup. It is currently in beta testing by some of our beta testers and things are looking good. Please give all credit to the author of this. Vlad is a gifted person!

**Comment by jbenzaquen42** [comment]
> Yup. It is currently in beta testing by some of our beta testers and things are looking good. Please give all credit to the author of this. Vlad is a gifted person!

All credit to Vlad for sure! I was expecting some extra wiring, but this looks much better than I expected!

**Comment by blinkitogaming** [comment]
+1 on this feature and really thanks to Vlad. Can't wait to see this working.
